### PR TITLE
Fixes #395 -- added next steps in docs for configuration of Open Zaak

### LINKED ISSUE #395: As sysadmin, I want to have instructions on how to configure Open Zaak for the first time
... so I can start working with after installation.

**Description**
Some documentation needs to be added on "next steps" (a link at the end of each deployment document) to configure Open Zaak for

- [x] Configure Sites.Site
- [ ] Configure basic API-access to eachother (so DRC can talk to ZRC, etc.) -- Maybe we need a button for this, to generate them as well?
- [ ] Configure notifications
- [ ] Create the first API token
- [ ] Make the first API call with this API token

References should be added to the manual for more details, but this document should summerize the first steps basically. Don't just copy text from manual to this doc.

### PR DESCRIPTION
**Pull request by stevenbal**
issue #395 

@joeribekker @sergei-maertens should this be in English or Dutch? Because I saw that the manual is largely in Dutch, while everything in Installation docs is in English

### CODE REVIEW

**Comment by sergei-maertens** [comment]
End user documentation (manual) is in Dutch because that's the target audience, while technical docs is in English as devs aren't necessarily Dutch-speaking.

File: docs/installation/configuration.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+==================
+
+In the admin, under **Configuration > Sites**, make sure to change the existing
+``Site`` to the domain under which Open Zaak will be deployed (see :ref:`manual_configuration` for more
+information).
+
+Configure API-access between components
+=======================================
+
+To allow communication between the different APIs in Open Zaak, e.g. between Zaken and Catalogi, etc.,
```
Comment (sergei-maertens) [inline docs/installation/configuration.rst]: Don't think this is correct, will double check tomorrow

File: docs/installation/deployment/single_server.rst [hunk 2]
```diff
@@ -230,3 +230,9 @@ services already listening on that port.
 
 .. _docker hub: https://hub.docker.com/u/openzaak
 .. _django secret key generator: https://miniwebtool.com/django-secret-key-generator/
+
+Next steps
+^^^^^^^^^^
```
Comment (sergei-maertens) [inline docs/installation/deployment/single_server.rst]: Not sure if this needs another title type? `=====` or `------` should do IMO.

File: docs/installation/configuration.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
+.. _configuration:
```
Comment (joeribekker) [inline docs/installation/configuration.rst]: I typically use `_<parent dir>_<file>` as a format, so

```suggestion
.. _installation_configuration:
```

File: docs/installation/configuration.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+``Site`` to the domain under which Open Zaak will be deployed (see :ref:`manual_configuration` for more
+information).
+
+Configure API-access between components
+=======================================
+
+To allow communication between the different APIs in Open Zaak, e.g. between Zaken and Catalogi, etc.,
+follow these steps:
+
+- Create ``Client credentials`` for each component (**API Authorizations > Client credentials**)
```
Comment (joeribekker) [inline docs/installation/configuration.rst]: **Client credentials** was removed and is only accessible via **API Authorizations > Applicaties**

File: docs/installation/configuration.rst [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+(see :ref:`manual_configuration`).
+
+
+Create an API token
+===================
+
+Create another ``Client credential`` and also create an ``Application`` for it. Add an ``Authorization``
+with the scope ``catalogi.lezen`` to this ``Application``. These
+credentials will be used to make the first request to the Open Zaak APIs. After creating the
+``Application``, copy the JSON Web Token (JWT) shown under **Client credentials**.
```
Comment (joeribekker) [inline docs/installation/configuration.rst]: Add a warning that this JWT has limited validity and should not be used in real applications.

File: docs/installation/deployment/kubernetes.rst [hunk 6]
```diff
@@ -356,3 +356,9 @@ Run the ``apps.yml`` playbook using:
 .. _Kubernetes: https://kubernetes.io/
 .. _NGINX: https://www.nginx.com/
 .. _Ansible: https://www.ansible.com/
+
+Next steps
+^^^^^^^^^^
```
Comment (joeribekker) [inline docs/installation/deployment/kubernetes.rst]: This can be a ==== as well

**Pull request merged**